src/sentry/static/sentry/app/views/settings/components/forms/formField.jsx

```diff
@@ -330,7 +329,7 @@ class FormField extends React.Component {
               } else if (isSaved) {
                 return (
                   <SettingsIsSaved>
-                    <IconCheckmarkSm size="18" />
+                    <InlineSvg src="icon-checkmark-sm" size="18px" />
```

Comment:
Most browsers use a common sense approach when it comes to unit-less values as attributes, which is why all these naked numbers get converted to pixels and not ems, percentages, etc. 

But that kind of ambiguity is confusing, unpredictable, and could potentially cause bugs. So I converted them all to pixel values.

Reply:
Makes sense.